ui/00_message/common/item_name.xml

```diff
@@ -19659,7 +19659,7 @@
   </item>
   <item id="7839">
     <original>鬼どぶろく</original>
-    <translation>Demon Unrefined Wine</translation>
+    <translation>Unrefined Demon Wine</translation>
```

Comment:
Not that it's the first mistranslation I've seen, but I don't really have all the time to go through and catch all of them haha